File: service/src/main/java/com/coopra/service/RetrofitHelper.java
package com.coopra.service;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitHelper {
    public static Retrofit createRetrofit() {
        return new Retrofit.Builder()
                .baseUrl("https://api.soundcloud.com")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }
}

File: service/src/main/java/com/coopra/service/RetrofitHelper.kt
package com.coopra.service

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

class RetrofitHelper {
    fun createRetrofit(): Retrofit {
        return Retrofit.Builder()
                .baseUrl("https://api.soundcloud.com")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
    }
}